functions/sheets.js

```diff
@@ -73,10 +76,17 @@ readSpreadsheetImpl = async function(competition, spreadsheetId) {
           return
         }
         var identifierVal = header.get(row)
+        if (!identifierVal) {
+          return
+        }
+        identifierVal = identifierVal instanceof String ? identifierVal.toUpperCase() : identifierVal
```

Comment:
I had to use this "trick" to deal with non-string identifiers (ie: wcaUserId).

Reply:
Ah, good call. I wasn't using non-string identifiers when I did this (only email, wcaId, name).